packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,202 +24,274 @@ _ref:
             The global object can also be modified using the [`SetGlobal`](/SetGlobal) action.
 
-            # Menus
-
-            Menu objects are an object with links to pages. The are used by blocks like `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+            # Connections
 
-            Blocks like like `PageSiderMenu` use the menu with `id: default` by default. This means that if you create a menu object with `id: default`, this will be used unless another menu is configured.
+            In a Lowdefy app you can integrate with other services like API's or databases using `connections` and `requests`. Connections configure the connection settings to the service, and often contain parameters like connection strings, urls and secrets such as passwords or API keys. Requests are used to interact with the connection, for example inserting a data record, executing a query or calling a API end-point.
 
-            More than one menu can be configured in an app. As an example, this can be used when two logically different sections in the app need different menus.
+            See more about how `connections` and `requests` are used together [here](/connections-and-requests).
 
-            If OpenID Connect authentication and role based authentication is configured, pages that the user is restricted from seeing are filtered from the menu object.
+            # Plugins
 
-            The schema for a menu object is:
+            TODO: Add plugin things here
 
-            - `id: string`: __Required__ - A identifier for the menu. If it is `default`, it will be used as default by pages.
-            - `links: object[]`: An array of `MenuLink` or `MenuGroup` objects that form the links in the menu. `MenuGroups` can be two levels deep.
+            # Menus
 
-            The schema for a `MenuLink` is:
-            - `id: string`: __Required__ - A identifier for the link unique to the menu.
-            - `type: string`: __Required__ - The type should be `MenuLink`.
-            - `pageId: string`: The id of the page to link to. Used as the title if no title is provided.
-            - `url: string`: An external url to link to.
-            -  `properties: object`: The properties of the menu link. These are:
-                - `title: string`: The title to display for the link.
-                - `icon: string | object`: The name of an [Ant Design Icon](https://ant.design/components/icon/) or properties of an Icon block to use as the icon for the link. The icon is only shown if the link is at the top level of the menu (not in a menu group).
+            Menu objects are objects with links to pages. They are used by blocks like `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
```

Comment:
Maybe we can do something like:

Menu objects are objects with links to pages. They are filtered to only show pages the user is authorised to see and are used by blocks like `PageSiderMenu` to render the links in the menu.

Reply:
> Menu objects are objects with links to pages. Menus are filtered automatically to only show pages the user is authorised to see based on [app and profile roles](TODO: link to roles page). Some blocks, like PageSiderMenu, will render the links in the menu.

Not sure it reads correctly, just think a reference to roles needs to be in there.